components/Hero/Hero.js

```diff
@@ -1,14 +1,25 @@
 import React from 'react'
+import { Wave } from './Wave'
 
 export function Hero() {
   return (
-    
-    <div className="hero float-end m-4 w-100">
-      <div className="hero__content pe-4 text-end">
-        <p>Hello I{`'`}m</p>
-        <h1>Abla Lucia Gonzalez</h1>
+   <div className="content__hero mb-4">
```

Comment:
When doing BEM naming the name of the component/block should come first "hero__content--blue"

http://getbem.com/naming/